ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -359,83 +359,47 @@ def test_torch_transpose(
     )
 
 
-# squeeze
+    
+# chunk
 @handle_cmd_line_args
 @given(
     dtype_and_values=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
-        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)),
+        ),
+        num_arrays=st.shared(helpers.ints(min_value=1,max_value=4), key="num_arrays"),
+        shape=st.shared(helpers.get_shape(min_num_dims=1,max_num_dims=4), key="shape"),
     ),
     dim=helpers.get_axis(
         shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
-        max_size=1,
     ).filter(lambda axis: isinstance(axis, int)),
     num_positional_args=helpers.num_positional_args(
-        fn_name="ivy.functional.frontends.torch.squeeze"
+        fn_name="ivy.functional.frontends.torch.chunk"
     ),
 )
-def test_torch_squeeze(
+    
+def test_torch_chunk(
     dtype_and_values,
     dim,
     as_variable,
     with_out,
     num_positional_args,
     native_array,
-    fw,
-):
-    input_dtype, value = dtype_and_values
-    helpers.test_frontend_function(
-        input_dtypes=input_dtype,
-        as_variable_flags=as_variable,
-        with_out=with_out,
-        num_positional_args=num_positional_args,
-        native_array_flags=native_array,
-        fw=fw,
-        frontend="torch",
-        fn_tree="squeeze",
-        input=np.asarray(value, dtype=input_dtype),
-        dim=dim,
-    )
 
-
-# swapaxes
-@handle_cmd_line_args
-@given(
-    dtype_and_values=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
-        shape=st.shared(helpers.get_shape(min_num_dims=2), key="shape"),
-    ),
-    axis0=helpers.get_axis(
-        shape=st.shared(helpers.get_shape(min_num_dims=2), key="shape"),
-    ).filter(lambda axis: isinstance(axis, int)),
-    axis1=helpers.get_axis(
-        shape=st.shared(helpers.get_shape(min_num_dims=2), key="shape"),
-    ).filter(lambda axis: isinstance(axis, int)),
-    num_positional_args=helpers.num_positional_args(
-        fn_name="ivy.functional.frontends.torch.swapaxes"
-    ),
-)
-def test_torch_swapaxes(
-    dtype_and_values,
-    axis0,
-    axis1,
-    as_variable,
-    with_out,
-    num_positional_args,
-    native_array,
-    fw,
 ):
-    input_dtype, value = dtype_and_values
+    input_dtype,value = dtype_and_values 
     helpers.test_frontend_function(
         input_dtypes=input_dtype,
         as_variable_flags=as_variable,
         with_out=with_out,
         num_positional_args=num_positional_args,
         native_array_flags=native_array,
         fw=fw,
-        frontend="torch",
-        fn_tree="swapaxes",
-        input=np.asarray(value, dtype=input_dtype),
-        axis0=axis0,
-        axis1=axis1,
-    )
+        frontend='torch',
+        fn_tree='chunk',
+        input = np.asarray(value,dtype=input_dtype),
```

Comment:
args should follow frontend sequence and name, which is `input, chunks, dim`

Reply:
okay , just changed that , thanks so much

Reply:
Hi , I have had issues reverting back the changes I made to swapaxes and squeeze(accidentally deleted them while resolving conflicts)  ,however  I have committed all the changes you suggested to the test functions .